build/template-publish-packages-and-symbols.yaml

```diff
@@ -35,3 +35,9 @@ steps:
   inputs:
     SearchPattern: '${{ parameters.SymbolPublishWildcard }}'
     SymbolServerType: TeamServices
+
+- task: PublishSymbols@2
```

Comment:
Do we still need the PublishSymbols task above?

Reply:
Yes. The two tasks support two different types of files